.gitignore

```diff
@@ -0,0 +1,19 @@
+.svn/
+.hg/
+.hgtags/
+log
+*.py[cod]
+build
+doc/_build
+dist/
+pylint.egg-info/
+.tox
+*.sw[a-z]
+doc/features.rst
+pyve
+build-stamp
+debian/files
```

Comment:
Should these have leading slashes since they refer to a specific directory rather than any `debian` directory anywhere in the repository? I think this applies to some of the other directories listed too.